run-tests-with-build-integration/pom.xml

```diff
@@ -87,29 +88,14 @@
                         </configuration>
                     </execution>
                     <execution>
-                        <id>run-tests</id>
-                        <phase>test</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
                         <configuration>
                             <pomIncludes>
                                 <pomInclude>droolsjbpm-integration/*/pom.xml</pomInclude>
                             </pomIncludes>
                             <pomExcludes>
                                 <pomExclude>droolsjbpm-integration/drools-examples-android/roboguice-*.*xml</pomExclude>
                                 <pomExclude>droolsjbpm-integration/kie-maven-plugin-.*xml</pomExclude>
-
-                                <pomExclude>droolsjbpm-integration-tests/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/kie-karaf-itests/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/drools-simulator/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/kie-maven-plugin-tests/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/test-suite/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/drools-examples-android/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/kie-maven-plugin-example/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/kie-maven-project-example/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/process-migration-service/pom.xml</pomExclude>
-                                <pomExclude>droolsjbpm-integration/kie-spring-boot/pom.xml</pomExclude>
+                                <pomExclude>droolsjbpm-integration/kie-osgi/kie-karaf-itests/pom.xml</pomExclude>
```

Comment:
Not sure what this is. We've removed execution id, phase and goal - what is the purpose?

Reply:
I need to check what's the default goal for a single execution, and why other modules don't have these configs.

Reply:
restored the id and goal tags for the second execution + indentation formatting in integration module